app/views/agency/uscis/index.html.erb

```diff
@@ -42,6 +42,9 @@
       </div>
       <div class="sm-col-right">
         <form action="/auth/saml" method="GET">
+          <% if params[:loa] %>
+            <%= hidden_field_tag 'loa', params[:loa] %>
+          <% end %>
```

Comment:
because the way these agency mockups have been created is non-standard, and because we'll be pulling them out of our eventual reference app, I made the decision to only convert the ones that used the standard layout, as referenced in the PR.